src/resources/views/components/modals/edit-profile.blade.php

```diff
@@ -1,14 +1,16 @@
 <div onclick="hideProfile()" id="profile"
     class="fixed left-0 top-0 bg-mydark bg-opacity-50 w-full h-full justify-center items-center opacity-0 hidden transition-opacity duration-500">
     <div onclick="event.stopImmediatePropagation()" class="bg-mycream rounded-2xl shadow-md p-10 flex">
+        <form method="POST" action="/update-profile">
```

Comment:
hmm update profile but the method is POST?

Reply:
+1

Reply:
also use route() with name when calling end points